### PR TITLE
feat(batch,bench): add microbench for batch hash agg

### PR DESCRIPTION
**Pull request by richardchien**
Signed-off-by: Richard Chien <stdrc@outlook.com>

I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

Add a simple micro benchmark for batch `HashAggExecutor`.

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

File: src/batch/benches/hash_agg.rs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+fn mock_task_id() -> TaskId {
+    TaskId {
+        task_id: 0,
+        stage_id: 0,
+        query_id: String::new(),
+    }
+}
+
+fn create_hash_agg_executor(chunk_size: usize, chunk_num: usize) -> BoxedExecutor {
```
Comment (liurenjie1024) [inline src/batch/benches/hash_agg.rs:40]: Please add more data types for grouping and agg functions, and make it easy to extend.

File: src/batch/benches/hash_agg.rs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    HashAggExecutorBuilder::deserialize(
+        &agg_node,
+        Box::new(child),
+        mock_task_id(),
+        "HashAggExecutor".to_string(),
+    )
+    .unwrap()
+}
+
+async fn executr_hash_agg_executor(executor: BoxedExecutor) {
```
Comment (liurenjie1024) [inline src/batch/benches/hash_agg.rs:70]:
```suggestion
async fn execute_hash_agg_executor(executor: BoxedExecutor) {
```

File: src/batch/src/executor/hash_agg.rs [hunk 3]
```diff
@@ -62,7 +62,7 @@ pub struct HashAggExecutorBuilder {
 }
 
 impl HashAggExecutorBuilder {
-    fn deserialize(
+    pub fn deserialize(
```
Comment (liurenjie1024) [inline src/batch/src/executor/hash_agg.rs:65]: Please follow other's convention to make `new` method public rather others.

File: src/batch/benches/hash_agg.rs (hunk 1 above)
Comment (richardchien) [reply src/batch/benches/hash_agg.rs]: Does it look better now?

**Review by liurenjie1024 — APPROVED** [review]
LGTM

**Pull request merged**